### ISSUE TITLE
Missing 'a' in cleaned .srt file

### ISSUE THREAD

**Issue by Pochoxo** [meizhong986/WhisperJAV#1]
First off, this is great and thank you very much for creating this!  It's much faster and streamlined than the WhisperWithVAD colab that many of us were using before!

One issue I'm noticing is that in the cleaned.srt copy, the 'a' in sentences will get removed.  I'm assuming this is part of the global parameters and trying to clean-up the srt but for example, in the sentence: "I'm going to have a chat."  The original .srt file will show the full sentence but the cleaned.srt will show "I'm going to have  chat."  The 'a' is removed and there is a double space in it's place.

Thank you again for providing this and helping the community!

**Comment by meizhong986** [comment]
Good catch!
It's a bug. I should correct it.

Happy subbing

**Comment by dgoryeo** [comment]
@Pochoxo , this needs a bit more digging. Just looked into few subs and didn't see any missing "a" in those. Example: 
```

3
00:00:29,340 --> 00:00:30,420
It's a nice place.

4
00:00:32,540 --> 00:00:34,420
There is a place called Kunji Bridge over there.

6
00:00:36,120 --> 00:00:37,660
It's a place with a great view.

8
00:00:39,700 --> 00:00:41,280
It's a strange name.

```

**Comment by Pochoxo** [comment]
@dgoryeo Hopefully this helps... I just used V0.6 and processed the sub for VENX-127 and here is a comparison between the two files (both included in the zip file since .srt wasn't supported for upload).  I'm assuming the RAW-Whipser-clean file is the good one that's been 'processed'.  You can see from the example that the 'a' in 'It's been a long time.' in Sub #7 is removed.  Also, the 'I'm sorry' in Sub #8 & Sub #9 are gone.  Finally, "in" from Sub #12 in the sentence "So, I thought you might be in trouble." is also removed.  I noticed this the most with 'a', 'in' and 'I'm sorry".  What I've been doing to find these manually is to to CTRL+F the file and find "  " (double-space gaps) and compare the two files for those subtitle timeframes.  Hope this helps!

**From the VENX-127-en RAW-Whipser-clean file:**
7
00:00:41,680 --> 00:00:43,180
It's been  long time.


8
00:00:43,540 --> 00:00:45,520
to come suddenly.


9
00:00:45,520 --> 00:00:46,000
I'm late.


10
00:00:47,380 --> 00:00:48,740
What's the matter?


11
00:00:50,480 --> 00:00:54,880
Kaori is going back to her parents' house.


12
00:00:56,060 --> 00:00:58,360
So, I thought you might be  trouble.


13
00:01:00,420 --> 00:01:02,640
I thought I could help you with the housework.


14
00:01:04,040 --> 00:01:05,640
Oh, I see.


15
00:01:06,580 --> 00:01:07,860
to bother you.


16
00:01:09,180 --> 00:01:09,700
It's okay.


17
00:01:10,340 --> 00:01:11,720
I have to go home tomorrow.


18
00:01:11,720 --> 00:01:15,300
I know I shouldn't ask you this, but I'll help you with anything.


19
00:01:16,350 --> 00:01:16,980
Thank you.


20
00:01:18,300 --> 00:01:19,780
Please come in.


21
00:01:20,380 --> 00:01:21,980
I'll carry your luggage.

**From the VENX-127-en.srt file found in the .subs.original folder:**
7
00:00:41,680 --> 00:00:43,180
It's been a long time.

8
00:00:43,540 --> 00:00:45,520
I'm sorry to come suddenly.

9
00:00:45,520 --> 00:00:46,000
I'm sorry I'm late.

10
00:00:47,380 --> 00:00:48,740
What's the matter?

11
00:00:50,480 --> 00:00:54,880
Kaori is going back to her parents' house.

12
00:00:56,060 --> 00:00:58,360
So, I thought you might be in trouble.

13
00:01:00,420 --> 00:01:02,640
I thought I could help you with the housework.

14
00:01:04,040 --> 00:01:05,640
Oh, I see.

15
00:01:06,580 --> 00:01:07,860
I'm sorry to bother you.

16
00:01:09,180 --> 00:01:09,700
It's okay.

17
00:01:10,340 --> 00:01:11,720
I have to go home tomorrow.

18
00:01:11,720 --> 00:01:15,300
I know I shouldn't ask you this, but I'll help you with anything.

19
00:01:16,350 --> 00:01:16,980
Thank you.

20
00:01:18,300 --> 00:01:19,780
Please come in.

21
00:01:20,380 --> 00:01:21,980
I'll carry your luggage.






[VENX-127-en RAW-Whipser-cleaned.zip](https://github.com/meizhong986/WhisperJAV/files/13085738/VENX-127-en.RAW-Whipser-cleaned.zip)

**Comment by meizhong986** [comment]
This was very helpful. 
I have just released version 0.6b that fixes the bug. Please take a look. I've tested it on three test cases and it worked well. Hope it is robust. Let me know.

**Comment by Pochoxo** [comment]
I just tried it and the output looks great!  I'll let you know if I run into the issue again or see anything else but so far it looks perfect.  Thank you once again for doing this and providing it to the community!  Awesome work!!
